packages/web-forms/src/components/common/map/useMapBlock.ts

```diff
@@ -256,7 +256,33 @@ export function useMapBlock(mode: Mode, events: { onFeaturePlacement: () => void
 
 	const teardownMap = () => {
 		mapViewControls?.stopWatchingCurrentLocation();
+		mapViewControls = undefined;
 		mapInteractions?.teardownMap();
+		mapInteractions = undefined;
+
```

Comment:
Maybe for consistency, it can also clean useMapFeatures, in case it has something that should be cleared at some point.
```suggestion
useMapFeatures = undefined;
```